### PR TITLE
Add rough draft of Creating a Pipeline page

### PR DESCRIPTION
**Pull request by tkyi**
- assuming a Screwdriver CLI exists in go

Related to documentation that needs to be done for https://github.com/screwdriver-cd/guide/issues/10

### CODE REVIEW

File: docs/getting-started/create-pipeline.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+## Set up Screwdriver CLI
+
+**The Screwdriver CLI does not exist yet, but here is what the instructions would looks like.**
+
+### Installing the CLI
+
+```bash
+$ go get github.com/screwdriver-cd/client
+$ $GOPATH/bin/client
```
Comment (Filbird) [inline docs/getting-started/create-pipeline.md:40]: Maybe mention something about ensuring `$PATH`  includes `$GOPATH/bin` :

`export PATH=$PATH:$GOPATH/bin`

Comment (FenrirUnbound) [inline docs/getting-started/create-pipeline.md:40]: Where would that set of instructions best fit in?

Comment (stjohnjohnson) [inline docs/getting-started/create-pipeline.md:40]: Wouldn't it be something like curl the release? @jer

File: docs/getting-started/create-pipeline.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Creating a Pipeline
+Using the Screwdriver CLI, you can build pipelines locally on your machine.
+
+## Set up Screwdriver CLI
+
+**The Screwdriver CLI does not exist yet, but here is what the instructions would looks like.**
```
Comment (petey) [inline docs/getting-started/create-pipeline.md]: `would look like`

File: docs/getting-started/create-pipeline.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+```bash
+$ sd repl
+```
+
+### Create a Pipeline
+To create a pipeline, you will need your `scmUrl`.
+```bash
+sd$ pipelines create {YOUR_SCMURL}  # (ex: git@github.com:screwdriver-cd/hashr.git#master)
+```
+Congratulations! You just ran your first build!
```
Comment (FenrirUnbound) [inline docs/getting-started/create-pipeline.md]: I don't believe that the `pipeline create` actually runs the build. It creates the job though

File: docs/getting-started/create-pipeline.md (hunk 1 above)
Comment (FenrirUnbound) [inline docs/getting-started/create-pipeline.md:40]: Setting up Go sounds like it's beyond the scope of this document though

**Comment by FenrirUnbound** [comment]
👍 

Looks good to me

fyzmeya - imgur

File: docs/getting-started/create-pipeline.md (hunk 1 above)
Comment (stjohnjohnson) [inline docs/getting-started/create-pipeline.md:40]: Yeah, I don't think we should be installing Go, we should be installing the client.  Example: 

``` bash
$ wget https://github.com/screwdriver-cd/client/releases/download/v1.0.2/sd
$ ./sd repl
```

**Comment by stjohnjohnson** [comment]
LGTM otherwise 👍

File: docs/getting-started/create-pipeline.md (hunk 1 above)
Comment (FenrirUnbound) [inline docs/getting-started/create-pipeline.md:40]: Discussed this during stand-up. We intend to keep this as-is, until we have a better alternative to how to convey this information.

**Pull request merged**